tests/test_pagination.py

```diff
@@ -249,3 +249,33 @@ def test_duplicates(self):
             next_url = self.pagination.get_next_link()
 
         self.assertIsNone(next_url)
+
+
+class TestSequenceSlicing(TestCase):
```

Comment:
These tests don't make sense in this file, they don't have anything to do with pagination, they're just testing part of the core `QuerySetSeqeuence` logic.

I'm unsure these tests really cover anything *different* than what the standard tests cover though...